src/helpers/tweets.validation.utils.ts

```diff
@@ -0,0 +1,12 @@
+import { ObjectId } from "mongodb";
+
+export const idValidator = (id: string) => {
+  if (!ObjectId.isValid(id)) {
+    throw new Error("Input error: incorrect Id");
+  }
+};
+export const stringValidator = (data: string) => {
+  if (data.trim().length === 0) {
+    throw new Error("Input error: password must not be empty");
+  }
```

Comment:
why function with name stringValidator throw password error?

function name does not match functionality

Reply:
fixed error message, probably deleted idea, will check it later in develop or delete it from there right after merge